File: scratch/check_layouts.py
import re

files = ['index.html', 'granular_media_downloads.html', 'smart_content_sorting.html']

for file in files:
    print(f"=== ANALYZING {file} ===")
    with open(file, 'r', encoding='utf-8') as f:
        content = f.read()
    
    # Find inline styles with flex, grid, width, margin
    styles = re.findall(r'style="([^"]*)"', content)
    for style in styles:
        if 'grid' in style or 'flex' in style or 'width' in style:
            # Print if it has fixed px values or large minmax
            if 'px' in style or 'minmax' in style:
                print(style)

File: scratch/check_layouts.js
const fs = require('fs');

const files = ['index.html', 'granular_media_downloads.html', 'smart_content_sorting.html'];

files.forEach(file => {
    console.log(`=== ANALYZING ${file} ===`);
    try {
        const content = fs.readFileSync(file, 'utf8');
        
        // Find style tags and inline styles
        const inlineStyles = content.match(/style="([^"]*)"/g) || [];
        inlineStyles.forEach(styleAttr => {
            const style = styleAttr.match(/style="([^"]*)"/)[1];
            if (style.includes('grid') || style.includes('flex') || style.includes('width') || style.includes('padding') || style.includes('margin')) {
                if (style.includes('px') || style.includes('minmax')) {
                    console.log(`  Inline: ${style}`);
                }
            }
        });
    } catch (err) {
        console.error(err);
    }
});
